amplipi/streams.py

```diff
@@ -1373,13 +1399,43 @@ def wait_on_proc(self):
       time.sleep(0.3)  # handles mock case
     self.state = 'stopped'  # notify that the audio is done playing
 
+  def send_cmd(self, cmd):
+    if cmd in self.supported_cmds:
+      if cmd == 'next':
+        if self._is_running():
+          self.proc.kill()
+          if self.bkg_thread:
+            self.bkg_thread.join()
+        self.proc = None
+      if self.command_file_path is not None:
+        if cmd == 'pause':
+          logger.info('PAUSE')
+          f = open(self.command_file_path, 'w')
+          f.write('pause')
+          f.close()
+          self.state = 'paused'
+
+        if cmd == 'play':
+          if not self._is_running():
+            logger.info(f'running: {self.vlc_args}')
+            self.proc = subprocess.Popen(args=self.vlc_args, preexec_fn=os.setpgrp)
+          f = open(self.command_file_path, 'w')
+          f.write('play')
+          f.close()
+          self.state = 'playing'
+
   def disconnect(self):
     if self._is_running():
       self.proc.kill()
       if self.bkg_thread:
         self.bkg_thread.join()
-    self.proc = None
     self._disconnect()
+    self.proc = None
+
+  def _is_running(self):
+    if self.proc:
+      return self.proc.poll() is None
+    return False
```

Comment:
Why is this duplicated from the base class?

Reply:
This function originally had additional behavior, and removing that left it with a copy of the base class version. Removing.